server/app/views/admin/programs/ProgramIndexViewV2.java

```diff
@@ -139,81 +154,176 @@ public ProgramDefinition getDisplayProgram(
   }
 
+  private Tag renderProgramRow(
+      boolean isActive,
+      ProgramDefinition program,
+      List<Tag> actions,
+      List<Tag> extraActions,
+      String... extraStyles) {
+    String badgeText = "Draft";
+    String badgeBGColor = BaseStyles.BG_CIVIFORM_PURPLE_LIGHT;
+    String badgeFillColor = BaseStyles.TEXT_CIVIFORM_PURPLE;
+    String updatedPrefix = "Edited on ";
+    Optional<Instant> updatedTime = program.lastModifiedTime();
+    if (isActive) {
+      badgeText = "Active";
+      badgeBGColor = BaseStyles.BG_CIVIFORM_GREEN_LIGHT;
+      badgeFillColor = BaseStyles.TEXT_CIVIFORM_GREEN;
+      updatedPrefix = "Published on ";
+    }
+
+    String formattedUpdateTime =
+        updatedTime.isPresent() ? renderDateTime(updatedTime.get(), zoneId) : "unknown";
+
+    int blockCount = program.getBlockCount();
+    int questionCount = program.getQuestionCount();
+
+    String extraActionsButtonId = "extra-actions-" + program.id();
+    ContainerTag extraActionsButton =
+        makeSvgTextButton("", Icons.MORE_VERT_PATH)
+            .withId(extraActionsButtonId)
+            .withClasses(
+                AdminStyles.TERTIARY_BUTTON_STYLES,
+                ReferenceClasses.WITH_DROPDOWN,
+                Styles.H_12,
+                extraActions.size() == 0 ? Styles.INVISIBLE : "");
+
+    return div()
+        .withClasses(
+            Styles.PY_7,
+            Styles.SPACE_X_10,
+            Styles.FLEX,
+            Styles.FLEX_ROW,
+            StyleUtils.hover(Styles.BG_GRAY_100),
+            StyleUtils.joinStyles(extraStyles))
+        .with(
+            p().withClasses(
+                    badgeBGColor,
+                    badgeFillColor,
+                    Styles.ML_8,
+                    Styles.FONT_MEDIUM,
+                    Styles.ROUNDED_FULL,
+                    Styles.FLEX,
+                    Styles.FLEX_ROW,
+                    Styles.GAP_X_2,
+                    Styles.PLACE_ITEMS_CENTER,
+                    Styles.JUSTIFY_CENTER)
+                .withStyle("min-width:90px")
+                .with(
+                    Icons.svg(Icons.NOISE_CONTROL_OFF_SVG_PATH, 20)
+                        .withClasses(Styles.INLINE_BLOCK, Styles.ML_3_5),
+                    span(badgeText).withClass(Styles.MR_4)),
+            div()
+                .with(
+                    p().with(
+                            span(updatedPrefix),
+                            span(formattedUpdateTime).withClass(Styles.FONT_SEMIBOLD)),
+                    p().with(
+                            span(String.format("%d", blockCount)).withClass(Styles.FONT_SEMIBOLD),
+                            span(blockCount == 1 ? " screen, " : " screens, "),
+                            span(String.format("%d", questionCount))
+                                .withClass(Styles.FONT_SEMIBOLD),
+                            span(questionCount == 1 ? " question" : " questions"))),
+            div().withClass(Styles.FLEX_GROW),
```

Comment:
did you run the formatter?  These newlines seem inconsistent

Reply:
I did run the formatter. The indentation seems fine to me, but I might be missing something.

Reply:
It's probably fine.  It's just that half of them have the div().withClass(classes) on the same line, and half split it up over 2 or 3 lines.